src/web/server.py

```diff
@@ -171,7 +174,10 @@ def wrapper(self, *args, **kwargs):
 
 
 def has_admin_rights(request_handler):
-    user_id = _identify_user(request_handler)
+    try:
+        user_id = _identify_user(request_handler)
+    except Exception:
```

Comment:
Could you logging for the error, please?

Reply:
Ok, but I found that it is legal case. /auth/info is not require authenticated user, but in case of requesting without authentication its gives 500.
May be it's just a not good place to fix and better to move catch to AuthInfoHandler?